src/rt-app.c

```diff
@@ -1401,13 +1401,22 @@ static void setup_main_gnuplot(void)
 			"plot ", tmp);
 
 		for (i=0; i<running_threads; i++) {
-			fprintf(gnuplot_script,
-				"\"%s-%s-%d.log\" u ($5/1000):4 w l"
-				" title \"thread [%s] (%s)\"",
-				opts.logbasename, threads[i].data->name,
-				threads[i].data->ind,
-				threads[i].data->name,
-				policy_to_string(threads[i].data->sched_data->policy));
+			if (opts.nthreads == opts.num_tasks) {
```

Comment:
There is no need to make a difference between these 2 cases because threads[i].data->ind is not needed in every case

Reply:
Ok, I thought they were two different cases.